### PR TITLE
(DED-634) Text API: Language detection  

### PR DESCRIPTION
**Pull request by kamilatokarska**
# Deploy preview

Scroll down to the checks section for the link to the deploy preview of your branch.

# Links

Link your Jira ticket.

- [Jira]()

# Description

Documentation for Text API & Language detection endpoints

# Review and release

Apply changes and resolve conflicts. Once the described functionality lands on prod, let us know on #platform-docs-releases that your PR is ready for release. We will **merge** and **publish** the changes.

### CODE REVIEW

File: src/pages/getting-started/index.mdx [hunk 1]
```diff
@@ -103,6 +103,10 @@ The Platform also provides you with a <a href="https://developers.livechat.com/d
 
 If you want to save time while building your app and don't want to play with APIs, the <Placeholder id="DEVELOPER_CONSOLE_URL"/> comes with a helping hand to provide easy access to all tools and resources for developers who want to build their apps. It is a place where you can configure your application, set up its most crucial functions, create all the necessary extensions, and display your app on the Marketplace.
 
+# Text API
+
+We want to provide you with a technology that allows you to build applications that meet all the customers' current needs and follow all the trends set by the world. Text API is open to the public and enables the building of powerful products using various Text operations. [Learn](/text-api/) what the Text Universe can already give you, and <a href="https://text.com/universe" target="_blank">take a look</a> at what we're about to offer soon.
```
Comment (ale-kacperczyk) [inline src/pages/getting-started/index.mdx:108]:
```suggestion
We want to provide you with a technology that allows you to build applications that meet all the customers' current needs and follow all the trends set by the world. Text API is open to the public and enables the building of powerful products using various text operations. [Learn](/text-api/) what the Text Universe can already give you, and <a href="https://text.com/universe" target="_blank">take a look</a> at what we're about to offer soon.
```

File: src/pages/text-api/index.mdx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+category: "text-api"
+title: "Text API Overview"
+tagline: "Learn about Text API."
+desc: "Learn about Text API and what powerful products you can build with it."
+slug: "/text-api/"
+---
+
+# Introduction
+
+<a href="https://text.com/universe" target="_blank">Text Universe</a> allows you to build powerful products using various Text operations like <a href="https://developers.livechat.com/data-and-reporting/" target="_blank">Text analytics</a> or <a href="https://developers.livechat.com/omnichannel/" target="_blank">Omnichannel APIs</a>. To make your experience even more prosperous and precious, we decided to open Text APIs to the public so you can freely discover, experiment, and build diverse applications.
```
Comment (ale-kacperczyk) [inline src/pages/text-api/index.mdx:12]:
```suggestion
<a href="https://text.com/universe" target="_blank">Text Universe</a> allows you to build powerful products using various text operations like <a href="https://developers.livechat.com/data-and-reporting/" target="_blank">Text analytics</a> or <a href="https://developers.livechat.com/omnichannel/" target="_blank">Omnichannel APIs</a>. To make your experience even more prosperous and precious, we decided to open Text APIs to the public so you can freely discover, experiment, and build diverse applications.
```

**Review by ale-kacperczyk — COMMENTED** [review]
We should also update the link here: 
image

and probably change the COMING SOON label to NEW (the same in the menu). You should be able to change it yourself in the code

DONE ✅

File: src/configs/redoc/text-api/spec.yml [hunk 3]
```diff
@@ -0,0 +1,330 @@
+openapi: 3.0.0
```
Comment (kamilatokarska) [inline src/configs/redoc/text-api/spec.yml:1]: @wfabjanczuk, here I pasted the content from the Customer Accounts API as it was easiest for me to check if everything works by doing it this way. Feel free to change it for the new one.

File: src/pages/text/index.mdx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+Part of the Text Universe is **Text API**, which currently offers language detection capabilities you can implement in your applications.
+
+And this is just a start. Soon, Text API will allow for more text-based operations, for example sentiment analysis or text clustering.
+
+## Language detection
+
+Language detection allows you to detect languages used in a message or a document. It can be very useful in all kinds of translator apps, advanced reporting, or auto replies.
+
+<SectionLink to={"/text/text-api#tag/Language-detection/"}>
```
Comment (oliwiapolec) [inline src/pages/text/index.mdx:24]:
```suggestion
<SectionLink to={"/text/text-api#tag/language-detection/"}>
```

Comment (oliwiapolec) [reply src/pages/text/index.mdx:24]: just a tiny suggestion :D

**Review by ale-kacperczyk — APPROVED** [review]

**Pull request merged**